dracut/30ignition/ignition-files.service

```diff
@@ -1,26 +1,28 @@
 [Unit]
 Description=Ignition (files)
 DefaultDependencies=false
-Before=initrd-parse-etc.service
-
-Requires=initrd-root-fs.target
-After=initrd-root-fs.target
+# This isn't the /etc you're looking for.
+#Before=initrd-parse-etc.service
 
 #Requires=ignition-setup.service ignition-disks.service
 #After=ignition-setup.service ignition-disks.service
 
 Requires=ignition-disks.service
 After=ignition-disks.service
 
-Requires=remount-sysroot.service
-After=remount-sysroot.service
+Requires=ignition-remount-sysroot.service
+After=ignition-remount-sysroot.service
+
+After=ostree-prepare-root.service
+After=initrd-switch-root.target
+Before=initrd-switch-root.service
 
 # setup the root filesystem before we try do things like create users on it.
 #Requires=initrd-setup-root.service
 #After=initrd-setup-root.service
 
 # setup networking
-After=network.target
+#After=network.target
```

Comment:
Do we know why this was necessary?

Reply:
I think it was just to narrow things down to the bare essentials. Will test with it enabled.